sql_generators/glean_usage/templates/baseline_clients_last_seen_v1.query.sql

```diff
@@ -1,9 +1,5 @@
 {{ header }}
-
-{% raw %}
-{% if is_init() %}
-{% endraw %}
-
+{% raw %}{% if is_init() %}{% endraw %}
 SELECT
   CAST(NULL AS INT64) AS days_seen_bits,
```

Comment:
For this init query to produce the intended column order it should select `submission_date` as the first column and instead of `*` use `* EXCEPT (submission_date)`.